packages/vsphere-cpi/charts/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+name: vsphere-cpi
+version: 1.0.0
+appVersion: 1.0.0
+description: vSphere CPI driver
+sources:
+  - https://github.com/kubernetes/cloud-provider-vsphere
+icon: file://logo.svg
+maintainers:
+  - name: Rancher
+    email: caleb@rancher.com
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/namespace: kube-system
+  catalog.cattle.io/release-name: vsphere-cpi
+  catalog.cattle.io/os: linux
```

Comment:
Need `display-name`. 
Also  `catalog.cattle.io/scope: management` if I understand this feature correctly. Shouldn't be able to deploy it downstream.

Reply:
So this should change to downstream

Reply:
@PennyScissors correct me if I'm wrong, but this should be deployable within the local cluster and downstream clusters right? Since both local cluster and downstream cluster can be vSphere clusters

Reply:
If thats case then we don't add annotation at all.

Reply:
Addressed display name in PR, still need confirmation w.r.t. scope annotation

Reply:
I'm not really sure about this one. @bmdepesa any ideas?

Reply:
It should be deployable on both local and downstream clusters.